pkg/workflows/execution/results.go

```diff
@@ -0,0 +1,13 @@
+package execution
+
+import "github.com/smartcontractkit/chainlink-common/pkg/values"
+
+type Results interface {
+	GetResultForStep(string) (*Result, bool)
```

Comment:
Does the `Get` prefix add value here?
```suggestion
	ResultForStep(string) (*Result, bool)
```
https://go.dev/doc/effective_go#Getters